optimize/self-managed/optimize-deployment/migration-update/camunda-8/instructions.md

```diff
@@ -0,0 +1,118 @@
+- [Back up your Elasticsearch instance](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-snapshots.html) in case something goes wrong during the migration process. This is recommended, but optional.
+- Make sure that you have enough storage available to perform the migration. During the migration process it can be the case that up to twice the amount of the storage of your Elasticsearch data is needed. (Highly recommended)
+- Back up your `environment-config.yaml` and `environment-logback.xml` located in the `config` folder of the root directory of your current Optimize. (Optional)
+- Start the new Optimize version, as described in the [installation guide](../../install-and-start.md).
+- It is very likely that you configured the logging of Optimize to your needs and therefore you adjusted the `environment-logback.xml` in the `config` folder of the root directory of your **old** Optimize. You can now use the backed up logging configuration and put it in the `config` folder of the **new** Optimize to keep your logging adjustments. (Optional)
+
+### 2. Rolling update to the new Elasticsearch version
+
+You only need to execute this step if you want to update the Elasticsearch (ES) version during the update. In case the ES version stays the same, you can skip this step.
+
+The Elasticsearch update is usually performed in a rolling fashion. Read all about how to do the ES update in the general [Elasticsearch Update Guide](https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-upgrade.html) and consult the [rolling ugprade](https://www.elastic.co/guide/en/elasticsearch/reference/current/rolling-upgrades.html) guide of the ES documentation on how to conduct the rolling update. If you have a very simple setup, for instance, a cluster with only one ES node without plugins installed nor machine learning jobs nor special configuration, the update would essentially boil down to the following steps:
+
+1. Install the new ES version, e.g. using Docker, your favorite package manager, or just by downloading and extracting the new tar/zip archive to a new directory.
+2. Copy the data from the old ES to the new ES. If you don't expect any new data coming to your old ES you can just copy the `data` folder from the old ES distribution and overwrite the `data` folder in the new ES distribution.
+3. Copy your old configuration (`config/elasticsearch.yml`) over to the new ES installation.
+4. Stop the old ES instance.
+5. Start the new ES instance and check that everything looks fine.
+
+Although the steps above summarize the basic update procedure, we recommend reading through the Elasticsearch documentation to avoid any potential issues.
+
+:::note Heads Up!
+
+Note that the following updates are not supported by Elasticsearch:
+
+- 6.8 to 7.0.
+- 6.7 to 7.1.–7.X (where X>1, e.g. 7.5)
+
+:::
+
+### 3. Perform the migration
```

Comment:
maybe we could instead refer to the helm chart docs for migration instructions? Although I think we do also have users not using helm charts for SM 🤔